src/api/OpenWith.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @flow
+ * @file Helper for the open_with_integrations API endpoint
+ * @author Box
+ */
+
+import Base from './Base';
+
+class OpenWith extends Base {
+    /**
+     * API URL for Open With
+     *
+     * @param {string} [id] - a box file id
+     * @return {string} base url for files
+     */
+    getUrl(id: string): string {
+        if (!id) {
+            throw new Error('Missing file id!');
+        }
+
+        return `${this.getBaseApiUrl()}/files/${id}/open_with_integrations`;
```

Comment:
Should this inherit from File.js?

Reply:
the getURL method in file assumes you're only passing in an ID, we would need to pass in a full suffix. We're not doing any other operations related to the file with this endpoint, so I think it's fine to leave as is.